app/webpacker/controllers/terms_and_conditions_controller.js

```diff
@@ -0,0 +1,34 @@
+import ApplicationController from "./application_controller";
+
+export default class extends ApplicationController {
+  static targets = ["filename", "fileinput"];
+  static values = {
+    message: String,
+  };
+
+  connect() {
+    super.connect();
+    this.fileinputTarget.addEventListener("change", (event) => {
+      this.filenameTarget.innerText = event.target.files[0].name;
+    });
+  }
+
+  remove(event) {
+    let confirmation = confirm(this.messageValue);
+    if (confirmation) {
+      location.hash = "";
+      this.stimulate(
+        "EnterpriseEdit#remove_terms_and_conditions",
+        event.target
+      );
+    }
+  }
+
+  stopEvent(event) {
+    event.preventDefault();
+  }
```

Comment:
I can't find any references to this method, is it a built-in name?

Reply:
Good catch, removed!